docker-app/qfieldcloud/core/tests/utils.py

```diff
@@ -51,13 +51,16 @@ def setup_subscription_plans():
 
 
 def set_subscription(
-    users: Union[User, Iterable[User]],
+    some_user: User | Iterable[User],
     code: str = None,
     **kwargs,
 ):
-    if isinstance(users, User):
-        users = [users]
+    users: list[User] = []
 
+    if isinstance(some_user, User):
+        users.append(some_user)
+    else:
+        users += some_user
     assert len(
```

Comment:
could it be:

users: list[User] = [some_user] if isintance(some_user, User) else some_users

BTW i would rather keep the var name `users` because it is simpler?

Reply:
Yes that works too!